addon/components/paper-form-container.js

```diff
@@ -0,0 +1,20 @@
+import Ember from 'ember';
+
+export default Ember.Component.extend({
+	tagName: '',
+	attributeBindings: ['style'],
+	style: "width:100%",
```

Comment:
We're only using single quotes. This should lead to a failing jscs error in the tests.